addons/ctrl-enter-post/addon.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "Ctrl+Enter to post",
+  "description": "Automatically posts the comment/post when you press Ctrl and Enter",
+  "credits": [
+    {
+      "name": "9gr",
+      "link": "https://scratch.mit.edu/users/9gr"
+    }
+  ],
+  "info": [
+    {
+      "type": "notice",
+      "text": "On macOS, you can use the Cmd key instead of the Ctrl key.",
+      "id": "macOS"
+    }
+  ],
+  "settings": [
+    {
+      "id": "forums",
+      "name": "Enable on forums",
```

Comment:
```suggestion
      "name": "Enable on forum threads",
```

Reply:
I think it's fine as is

Reply:
forums.js applies to edit post, topic reply and signature modal